src/FTPFilesystem.h

```diff
@@ -15,7 +15,7 @@ class FTPFilesystem
 	void addFilesystem(String Name, FS * const Filesystem);
 	void clearFilesystemList();
 
-	File open(const String & path, const char* mode = FILE_READ);
+	File open(const String & path, const char* mode = "r");
```

Comment:
is the `FILE_READ` not supported on ESP8266? I would like to stay with constants if possible.

Reply:
Unfortunately it is. I return constant